packages/api/src/controllers/stripe.ts

```diff
@@ -92,6 +113,36 @@ async function reportUsageForUser(req, user, adminToken) {
     subscription: user.stripeCustomerSubscriptionId,
   });
 
+  if (
+    user.stripeProductId.includes("hacker_1") ||
+    user.stripeProductId.includes("prod_O9XuIjn7EqYRVW")
+  ) {
+    // If they don't have a card and in overusage, send a notification email
+    // If they have a card and in overusage, report the usage
```

Comment:
Is this comment applicable? It seems like it doesn't send the usage for happy users that have a card, it just sends the payment failed email for all users instead.

Reply:
Yeah it's referring to the check below the comment, I moved it inside the scope of the condition